controllers/user.js

```diff
@@ -0,0 +1,93 @@
+import models from '../models';
+import crypto from 'crypto';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config()
+
+const { User } = models;
+class Users {
+
+    /**
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} Returns the User Object
+   */
+    static async createUser(req, res) {
+        const {
+            email, username, password: hash
+          } = req.body;
+
+          // check if the email exists
+          const userFind = await User.findOne({ where: { email: email } });
+          if(userFind) {
+            res.status(400).send({
+              status: 400,
+              errorMessage: 'the user with that email exists'
+            })
+          }
+
+          // if there is the server error that the user can not be saved
+          const user = await User.create( { email, username, hash } );
```

Comment:
When creating a local user we suggested to use salt and hash too on password creation but I see you missed out salt here

Reply:
The salt is generated itself before the password is saved it is mixed with it.

Reply:
The salt is generated itself before the password is saved it is mixed with it.